### PR TITLE
LIMs-354

### PR DESCRIPTION
**Pull request by dmoreno-harveyRoad**
This Pull request is great.

### CODE REVIEW

**Comment by Galithil** [comment]
CIrcle ci test do not pass, please fix

**Review by Galithil — COMMENTED** [review]
Starting review

File: README.md [hunk 1]
```diff
@@ -27,3 +27,5 @@ If you wish to build your own documentation, you must install sphinx_rtd_theme
 
 
 This is my edit.
+
```
Comment (Galithil) [inline README.md]: THis line doesnt fit in here

Comment (Galithil) [inline README.md]: I really don't agree with this

**Review by Galithil — CHANGES_REQUESTED** [review]
I can't agree with your changes please fix

File: README.md (hunk 1 above)
Comment (dmoreno-harveyRoad) [reply README.md]: OKay fine

Comment (dmoreno-harveyRoad) [reply README.md]: yeah yeah

**Comment by dmoreno-harveyRoad** [comment]
@Galithil I'm done

**Review by Galithil — APPROVED** [review]

**Pull request merged**